### DISCUSSION TITLE
Student Developer Pack approved but no benefit

### DISCUSSION THREAD

**Discussion by letrad** [community/community#57998] [GitHub Education]
### Hi everyone,

Hello,

I have recieved confirmation from the [relevant application page](https://education.github.com/discount_requests/application) that my application has been approved:
![image](https://github.com/community/community/assets/134692905/9416cdc9-dab6-4da6-9390-2ec7d3f5c392)

However, I have recieved no email from neither my primary, nor education email to state that I have recieved any benefits. Logging into my account shows no benefits whatsoever.

**Comment by jge162** [comment]
It takes 24-72 hours to show up. FYI, when I was approved it took 3 days.

**Comment by letrad** [comment]
Actually. it seems jst making this post got me accepted. Perhaps their automatic approval is messed up

**Reply by jge162** [reply]
Congrats

**Reply by jge162** [reply]
It’s just random, the time is anywhere between 24-72 hours. Wasn’t the post.

**Comment by businessservic** [comment]
> ### Hi everyone,
> Hello,
> 
> I have recieved confirmation from the [relevant application page](https://education.github.com/discount_requests/application) that my application has been approved: ![image](https://user-images.githubusercontent.com/134692905/245665061-9416cdc9-dab6-4da6-9390-2ec7d3f5c392.png)
> 
> However, I have recieved no email from neither my primary, nor education email to state that I have recieved any benefits. Logging into my account shows no benefits whatsoever.
> 
> ### A suggestion...(optional)
> _No response_

09078451759

**Reply by letrad** [reply]
???

**Comment by letrad** [comment]
The amount of bots is crazy

**Reply by AhmedKahlaoui** [reply]
Same issue, my application was approved 3 days ago but didn't receive any email afterwards. I don't know how much longer i need to wait in order to receive the benefits.
![image](https://github.com/community/community/assets/123409443/6d1a7ca5-694e-444d-ae67-3cabb75ee376)

**Comment by Raghavendrabaheti** [comment]
i received same problem and it was approved on 11nov but not show anything can anyone help  


However, I have recieved no email from neither my primary, nor education email to state that I have recieved any benefits. Logging into my account shows no benefits whatsoever.

**Comment by masabia** [comment]
I have the same problem , the request has been approved but I cannot access the benefits. Can anyone help?

**Comment by ghost** [comment]
I have the same problem. My request was approved on 21st Dec, but still I can't access benefits. :(

**Reply by drendev** [reply]
Did you get the benefits?

**Comment by YoavNahumus** [comment]
I got the same problem, I was approved back in Jan 20th and still no benefits available
![image](https://github.com/community/community/assets/62788945/06aecc04-9127-4fa8-a251-c240c5802dd8)

**Reply by devashish2024** [reply]
Bro what huh I jus got instant approve on apply ....

**Reply by YoavNahumus** [reply]
it still wasn't approved, been 2 months :(

**Reply by devashish2024** [reply]
I got approved in 4 or  5 days

**Reply by GustavoSchip** [reply]
Mine got approved yesterday, still waiting for the benefits.

**Reply by devashish2024** [reply]
I got benefts

**Reply by GustavoSchip** [reply]
> I got benefts

I still don't rip

**Reply by devashish2024** [reply]
lol oh i got 4 days later

**Comment by AsheCelesteLucia** [comment]
I'm still waiting for mine. The system is weird.

**Comment by atulyasndrm** [comment]
I am also having trouble accessing the benefits.

**Reply by mjpnet** [reply]
How long did it take finally ?  Im in the same situation, several days and no solution

**Comment by enesdouaydin** [comment]
![image](https://github.com/user-attachments/assets/5689a051-1d7a-4ec6-94ff-c60a126a3ad1)
i have a same problems.

**Comment by Juan-Clay** [comment]
I have waited for almost a month and still not recived the benifits after the website saying I was approved, who do I contact to get the benifits for my account?
